bin/bind.rs

```diff
@@ -1,25 +1,39 @@
-#![warn(missing_docs, unsafe_code)]
-
+#![warn(missing_docs)]
 use std::process::Command;
 
-pub(crate) fn bind_forge() -> std::io::Result<()> {
+use log::info;
```

Comment:
You have to add an [`env_logger`](https://docs.rs/env_logger/latest/env_logger/) or else logs can't be displayed.

Logs might be overkill here, actually. A print to console is probably fine when it is simple and we always want a user to see it.